code/modules/clothing/outfits/plasmaman.dm

```diff
@@ -143,8 +147,9 @@
 	uniform = /obj/item/clothing/under/plasmaman/atmospherics
 
 /datum/outfit/plasmaman/command
-	name = "Command Plasmamand"
-	
+	name = "Command Plasmaman"
+
+	mask = /obj/item/clothing/mask/gas/sechailer
```

Comment:
Why do they get a sechailer?

Reply:
he has one in his suit storage which most captains wear, and hes gotta wear an internals mask anyways